include/wget/wget.h

```diff
@@ -265,6 +265,8 @@ int
 	wget_global_get_int(int key) LIBWGET_EXPORT;
 void
 	(*wget_global_get_func(int key))(const char *, size_t) LIBWGET_EXPORT;
+void
+	wget_handle_sigwinch(void) LIBWGET_EXPORT;
```

Comment:
Currently, the idea was to provide a library function that acts as a signal handler for SIGWINCH. However, if we rename it to wget_bar_resize, we must ask the client to handle the signals and send libwget just a new width dimension.